### PR TITLE
Allow disable compat

### PR DESCRIPTION
**Pull request by kagg-design**
Commits by @WillPresley rebased to the current master. Tests added. Version bumped up.

### CODE REVIEW

**Comment by WillPresley** [comment]
@kagg-design This all looks great to me, I haven't tested the new testing suite but the plugin itself works as it should from this branch. Thanks again for rebasing & writing the tests and looking forward to the 1.8.0 release!

File: .tests/php/integration/Backend/SettingsTest.php [hunk 1]
```diff
@@ -403,6 +411,14 @@ public function test_hcap_display_options_page_update() {
 	<br><br>
 				<input
 					type="checkbox"
+					id="hcaptcha_recaptchacompat"
+					name="hcaptcha_recaptchacompat"
+				/>
+			&nbsp;
+			<span>Disable recaptchacompat (stop replacing other instances of reCAPTCHA with hCaptcha)</span>
```
Comment (e271828-) [inline .tests/php/integration/Backend/SettingsTest.php:418]:
```suggestion
			<span>Disable reCAPTCHA Compatibility Mode (stop replacing other instances of reCAPTCHA with hCaptcha)</span>
```

File: .tests/php/integration/Common/FunctionsTest.php [hunk 2]
```diff
@@ -90,6 +90,10 @@ public function test_hcap_options() {
 					'type'        => 'text',
 					'description' => 'Info on <a href="https://hcaptcha.com/docs/languages" target="_blank">language codes</a>.',
 				],
+			'hcaptcha_recaptchacompat'          => [
+				'label' => 'Disable recaptchacompat (stop replacing other instances of reCAPTCHA with hCaptcha)',
```
Comment (e271828-) [inline .tests/php/integration/Common/FunctionsTest.php:94]:
```suggestion
				'label' => 'Disable reCAPTCHA Compatibility (use if including both hCaptcha and reCAPTCHA on the same page)',
```

File: readme.txt [hunk 3]
```diff
@@ -71,6 +71,9 @@ Please see the hCaptcha privacy policy at:
  
 == Changelog ==
 
+= 1.8.0 =
+* Added option to disable recaptchacompat (stop replacing other instances of reCAPTCHA with hCaptcha).
```
Comment (e271828-) [inline readme.txt:75]:
```suggestion
* Added option to disable reCAPTCHA Compatibility (use if including both hCaptcha and reCAPTCHA on the same page)
```

**Review by e271828- — APPROVED** [review]

**Comment by WillPresley** [comment]
@e271828- @kagg-design Is there any reason all of this was dropped for the new releases?

**Comment by kagg-design** [comment]
@WillPresley I will return back all the new features in 1.9.0, soon.

**Pull request merged**